@minggo 
I check the build logs, there is no error for compile, mac platform faield with script broken: 

*\* BUILD SUCCEEDED **

Checking out files:   8% (53/630)  
...
Checking out files: 100% (630/630), done.
HEAD is now at a2c5304 Merge commit 'refs/pull/15558/head' of https://github.com/cocos2d/cocos2d-x into pull15558
Removing external/Box2D/
Removing external/ConvertUTF/
Removing external/bullet/
Removing external/chipmunk/
Removing external/clipper/
Removing external/curl/
Removing external/edtaa3func/
Removing external/flatbuffers/
Removing external/freetype2/
Removing external/glfw3/
Removing external/jpeg/
Removing external/json/
Removing external/linux-specific/
Removing external/lua/
Removing external/png/
Removing external/poly2tri/
Removing external/recast/
Removing external/spidermonkey/
Removing external/sqlite3/
Removing external/tiff/
Removing external/tinyxml2/
Removing external/unzip/
Removing external/version.json
Removing external/webp/
Removing external/websockets/
Removing external/win10-specific/
Removing external/win32-specific/
Removing external/winrt_8.1-specific/
Removing external/wp_8.1-specific/
Removing external/xxhash/
Removing external/xxtea/
Removing external/zlib/
Removing tests/js-tests/res/
Removing tests/lua-empty-test/src/cocos/
Removing tests/lua-tests/src/cocos/
Removing tools/fbx-conv/
Removing tools/tojs/userconf.ini
Removing tools/tolua/userconf.ini
Removing v3-deps-92.zip
Slave went offline during the build
ERROR: Connection was broken: java.io.IOException: Unexpected termination of the channel
    at hudson.remoting.SynchronousCommandTransport$ReaderThread.run(SynchronousCommandTransport.java:50)
Caused by: java.io.EOFException
    at java.io.ObjectInputStream$PeekInputStream.readFully(ObjectInputStream.java:2325)
    at java.io.ObjectInputStream$BlockDataInputStream.readShort(ObjectInputStream.java:2794)
    at java.io.ObjectInputStream.readStreamHeader(ObjectInputStream.java:801)
    at java.io.ObjectInputStream.<init>(ObjectInputStream.java:299)
    at hudson.remoting.ObjectInputStreamEx.<init>(ObjectInputStreamEx.java:48)
    at hudson.remoting.AbstractSynchronousByteArrayCommandTransport.read(AbstractSynchronousByteArrayCommandTransport.java:34)
    at hudson.remoting.SynchronousCommandTransport$ReaderThread.run(SynchronousCommandTransport.java:48)

Build step 'Execute Python script' marked build as failure
An attempt to send an e-mail to empty list of recipients, ignored.
Finished: FAILURE

--- a/tests/cpp-tests/Classes/NewRendererTest/NewRendererTest.cpp
+++ b/tests/cpp-tests/Classes/NewRendererTest/NewRendererTest.cpp
@@ -523,7 +523,7 @@ std::string CaptureNodeTest::subtitle() const
 }
 
 void CaptureNodeTest::onCaptured(Ref*)
-{
+{ 
     Director::getInstance()->getTextureCache()->removeTextureForKey(_filename);
     removeChildByTag(childTag);
     

